### ISSUE TITLE
advance_type_casting regex fails to parse time like fields

### ISSUE THREAD

**Issue by townie** [savonrb/nori#70]
Currently not correctly parsing X-ContactStamp.

This appears to be an issue with the XS_TIME regex

``` ruby
> xml = "<InternetMessageHeader HeaderName=\"X-ContentStamp\">25:12:3380570386</InternetMessageHeader>"
> Nori.parse( xml)
RangeError: integer 3380570386 too big to convert to `int'
from .rbenv/versions/2.1.3/lib/ruby/2.1.0/time.rb:264:in `local'

backtrace
=> [".rbenv/versions/2.1.3/lib/ruby/2.1.0/time.rb:264:in `local'",
 ".rbenv/versions/2.1.3/lib/ruby/2.1.0/time.rb:264:in `make_time'",
 ".rbenv/versions/2.1.3/lib/ruby/2.1.0/time.rb:331:in `parse'",
 ".rbenv/versions/2.1.3/gemsets/backupify-clean/gems/nori-1.1.5/lib/nori/xml_utility_node.rb:223:in `block in advanced_typecasting'",
 ".rbenv/versions/2.1.3/gemsets/backupify-clean/gems/nori-1.1.5/lib/nori/xml_utility_node.rb:259:in `call'",
 ".rbenv/versions/2.1.3/gemsets/backupify-clean/gems/nori-1.1.5/lib/nori/xml_utility_node.rb:259:in `try_to_convert'",
 ".rbenv/versions/2.1.3/gemsets/backupify-clean/gems/nori-1.1.5/lib/nori/xml_utility_node.rb:223:in `advanced_typecasting'",
 ".rbenv/versions/2.1.3/gemsets/backupify-clean/gems/nori-1.1.5/lib/nori/xml_utility_node.rb:140:in `to_hash'",
 ".rbenv/versions/2.1.3/gemsets/backupify-clean/gems/nori-1.1.5/lib/nori/parser/nokogiri.rb:44:in `parse'",
 ".rbenv/versions/2.1.3/gemsets/backupify-clean/gems/nori-1.1.5/lib/nori/parser.rb:31:in `parse'",
 ".rbenv/versions/2.1.3/gemsets/backupify-clean/gems/nori-1.1.5/lib/nori.rb:12:in `parse'",
```

**Comment by tjarratt** [comment]
Thanks for reporting this issue. I don't have a lot of time to fix this right now, but if you'd care to issue a pull request I'd be more than happy to review and merge it. 

Sent From A Very Small Keyboard

**Comment by townie** [comment]
@tjarratt thanks for responding. We will be submitting a pull request for the fix. What is the policy around porting it fixes to older versions SavonRB, currently we are on 1.5 :(?

**Comment by tjarratt** [comment]
I don't recommend trying to backport it, but if you'd like to, and all the tests pass, I'm not opposed to releasing a small fix for a 1.5.x release. I do not think such a branch exists today, but there should be a tag. 

Sent From A Very Small Keyboard
